packages/types/src/mpe-server-to-temporal.ts

```diff
@@ -22,3 +22,27 @@ export const MpeAddTracksResponseBody = z.object({
     ok: z.literal(1),
 });
 export type MpeAddTracksResponseBody = z.infer<typeof MpeAddTracksResponseBody>;
+
+export const MpeChangeTrackOrderOperationToApply = z.enum(['UP', 'DOWN']);
+export type MpeChangeTrackOrderOperationToApply = z.infer<
+    typeof MpeChangeTrackOrderOperationToApply
+>;
+
+export const MpeChangeTrackOrderRequestBody = z.object({
+    workflowID: z.string(),
+    trackID: z.string(),
+    fromIndex: z.number().min(0),
+    userID: z.string(),
+    deviceID: z.string(),
+    operationToApply: MpeChangeTrackOrderOperationToApply,
+});
+export type MpeChangeTrackOrderRequestBody = z.infer<
+    typeof MpeChangeTrackOrderRequestBody
+>;
+
+export const MpeChangeTrackOrderResponseBody = z.object({
```

Comment:
I really like to have a type for each response body, even if they are all the same. If in the future the body changes, we won't have to change the type we are expecting the `got` response to be, but we will probably have to change what we do from the new body.
We know that the code using the response is likely to change, therefore it will be the first place where we will search for bugs. It's less true for the _type_ of a response body.